app.js

```diff
@@ -0,0 +1,158 @@
+var express 	= require('express');
+var fs 		    = require('fs');
+var router 		= express.Router();
+var path 		= require("path");
+var mongoose    = require('mongoose');
+var app         = express();
+var bodyParser  = require('body-parser');
+const methodOverride = require( 'method-override' );
+
+app.use(bodyParser.json() );
+
+try {var uristring = require('./data/mongolabinfo.js').loginstring;
+}
+catch(err){
+}
```

Comment:
What's up with the identation on this block? Also, since no valid db connection, you should throw an error instead of swallowing it in catch block.